src/main/java/org/spongepowered/api/entity/living/player/User.java

```diff
@@ -70,6 +72,33 @@
      */
     Optional<Player> getPlayer();
 
+    /**
+     * Gets the position of this User
+     *
+     * @return The position of this User
+     */
+    Vector3d getPosition();
+
+    /**
+     * Gets the World UUID of this User.
+     *
+     * <p>May return empty when the world the player is in does not exist anymore</p>
+     *
+     * @return The World UUID of this User if found
+     */
+    Optional<UUID> getWorldUUID();
```

Comment:
`getWorldUniqueId()` to match elsewhere in the API(`Identifiable`) and to fix checkstyle warning.